src/commands/git/worktree.ts

```diff
@@ -923,15 +950,18 @@ export class WorktreeGitCommand extends QuickCommand<State> {
 				} while (retry);
 			}
```

Comment:
Basically just changed
```suggestion

			if (branches.length > 0) {
				yield* getSteps(
					this.container,
					{
						command: 'branch',
						state: {
							subcommand: 'delete',
							repo: state.repo,
							references: branches,
						},
					},
					this.pickedVia,
				);
			}
```